README.md

```diff
@@ -275,6 +275,7 @@ The following npm scripts are available:
 * `npm test` - Runs tests.
 * `npm run check-fidelity` - Compare rendering to third-party renderers
 * `npm run install-renderers` - Install or update third-party renderers
+* `npm run fetch-samples` - Create or update a local copy of sample models. This is needed for running fidelity checks and updating screenshots. Warning: the samples are hundreds of megabytes large in aggregate!
```

Comment:
If this is required, does it make sense to merge it in with (perhaps) `install-renderers` (another long, expensive prep step) into a single script named something like `prepare-fidelity`?

Reply:
Or, if the thinking is that this is always run in `npm install`, maybe mention that (or leave it out entirely?)

Reply:
I guess for factorization purposes it would be nice to keep these things separated. For usability purposes, we probably want a single command.

It's easy enough for us to make a combined command and just not document the sub-commands.

I'm also happy to leave it out entirely.

Reply:
Agreed re factorization, I think particularly the backend scripts (the `.sh` or `.js` files) having those split and encapsulated helps to understand the code.

I like both a combined commands and undocumented sub-commands (I've used a `-` prefix to connotate internal sub-commands) or leaving that sub-commands out entirely.

The thing I like most is having the combined command for usability.

Reply:
Not a blocker to a merge, though.

Reply:
Added a command called `npm run bootstrap-fidelity-dev` that checks out samples and builds filament in one go. Removed docs for the individual fetch / install tasks.